tools/replay-viewer/style.css

```
@@ -759,6 +759,29 @@ button:disabled {
   box-shadow: 0 0 12px rgb(139 255 165 / 40%);
 }
 
+.legend-node {
+  width: 18px;
+  height: 18px;
+  border-radius: 999px;
+  background: #5cc8ff;
+  box-shadow: 0 0 14px rgb(92 200 255 / 42%);
+}
+
+.legend-node--primary {
```

Stylelint (17.13.0)

[error] Expected class selector ".legend-node--primary" to be kebab-case (selector-class-pattern)

(selector-class-pattern)